app/src/main/java/com/ac03/covid/ui/home/HomeFragment.kt

```diff
@@ -59,8 +60,20 @@ class HomeFragment : Fragment(R.layout.fragment_home) {
     }
 
     private fun updateUi(model: UiModel) {
+
+        if (model is Loading) {
```

Comment:
aqui mejor esto:

`private fun showProgres() = with(binding) {
        progress.visibility = View.VISIBLE
        layoutConfirmedCases.visibility = View.GONE
        layoutDeathCases.visibility = View.GONE
        layoutRecoveredCases.visibility = View.GONE
    }
    
    private fun hideProgress() = with(binding) {
        progress.visibility = View.GONE
        layoutConfirmedCases.visibility = View.VISIBLE
        layoutDeathCases.visibility = View.VISIBLE
        layoutRecoveredCases.visibility = View.VISIBLE
    }`
    
    en el is Loading llamas a showProgress, y en el isError y isContent llamas a hideProgress